pom.xml

```diff
@@ -14,8 +14,8 @@
 	<packaging>jar</packaging>
 
     <properties>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
why change it to java 8/why was it java 11 in the first place?

Reply:
Changed in maintenance, not sure why the actions were running - the dependencies here need java 8, we had this problem multiple times in the past :/

Reply:
what exactly is not working?

Reply:
nevermind, I confused two issues here